### PR TITLE
feat: add an operator

### PR DESCRIPTION
**Pull request by shon-button**
Addresses [1960](https://github.com/bcgov/cas-registration/issues/1960)

### 🚀 Impact:
- Updated `OperatorPage` for create action
- Added POST api `bc_obps/registration/api/v2/user_operators.py`
- Added service V2 `bc_obps/service/user_operator_service_v2.py`
- Added endpoint tests
- Added service tests
- Added vitests

### ✏️ Notes
1.  **e2e test:** Added to [tech debt ticket](https://github.com/bcgov/cas-registration/issues/2271)

### 🔬 Local Testing:

1. From terminal command, start the api server:
 ```
cd bc_obps
make reset_db
make run
```  
3.  From terminal command, start the app development server:
 ```
cd client && yarn dev-all

```

### Add an Operator:
1. Navigate to `http://localhost:3000`
2. Click button "Log in with Business BCeID"
4. Sign in to Keycloak using `bc-cas-dev-secondary`
**Expected results:** 
- Redirection to http://localhost:3000/dashboard page for role industry_user without an associated operator
- Administration tile has link `Select an Operator`
image
5. Click link `Administration\Select an Operator` 
**Expected results:** 
- Select an Operator page is displayed
- Page has link `Add Operator`
image
6. Click link `Add Operator`
**Expected results:** 
- Add an operator page displays
- Page has button `Submit`
- Page has button `Cancel`
image
7. Click button `Cancel`
**Expected results:** 
- Select an Operator page is displayed
8. Click link `Add Operator`
9. Click button `Submit`
**Expected results:** 
- Form does not submit, and displays required fields
image
10. Complete required fields
11. Click button `Submit`
**Expected results:** 
- Form submits and creates a new approved operator with an approved admin user operator
12. Click breadcrumb `Administration`
**Expected results:** 
- Administration tile has link `My Operator`
image
14. Click link `Administration\My Operator` 
**Expected results:** 
- Operator Information Details displayed data entered in the add operator form

### CODE REVIEW

File: bciers/apps/administration/app/components/operators/OperatorPage.tsx [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import { notFound } from "next/navigation";
 import OperatorForm from "./OperatorForm";
 import { operatorSchema } from "../../data/jsonSchema/operator";
```
Comment (marcellmueller) [inline bciers/apps/administration/app/components/operators/OperatorPage.tsx:2]: Tiniest of nits but I think moving up multiple directories like this in imports makes the code less readable. We can import from `apps/administration/data/jsonSchema/operator` and still avoid the circular import issue when using `@/administration` and the next person to read this file will know exactly where that import is coming from :)

**Review by marcellmueller — APPROVED** [review]
Nice work, this is working great and no issues I could find 🤓

File: bciers/apps/administration/app/components/operators/OperatorPage.tsx (hunk 1 above)
Comment (shon-button) [reply bciers/apps/administration/app/components/operators/OperatorPage.tsx:2]: @marcellmueller 
Totally agree! 

I didn't realize we could avoid the circular import with that technique- thanks :)

Comment (marcellmueller) [reply bciers/apps/administration/app/components/operators/OperatorPage.tsx:2]: To be clear I'm not sure if that is okay for shared imports outside of the project 🤔

**Pull request merged**